routingpy/routers/opentripplanner_v2.py

```diff
@@ -210,53 +178,47 @@ def _parse_legs(self, legs):
     def isochrones(
         self,
         locations: List[float],
-        profile: Optional[str] = "WALK,TRANSIT",
-        time: Optional[datetime.datetime] = datetime.datetime.now(datetime.timezone.utc),
-        cutoffs: Optional[List[int]] = [3600],
+        profile: str,
+        intervals: List[int],
+        interval_type: Optional[str] = None,
+        date_time: Optional[datetime.datetime] = datetime.datetime.now(datetime.timezone.utc),
+        date_time_type: Optional[str] = None,
         arrive_by: Optional[bool] = False,
         dry_run: Optional[bool] = None,
     ):
         """Gets isochrones for a range of time values around a given set of coordinates.
 
         :param locations: Origin of the search as [lon,lat].
-        :type locations: list of float
-
-        :param profile: Comma-separated list of transportation modes that the user is willing to
-            use. Default: "WALK,TRANSIT"
-        :type profile: str
-
-        :time: Departure date and time (timezone aware). The default value is now (UTC).
-        :type time: datetime.datetime
-
-        :cutoff: The maximum travel duration in seconds. The default value is one hour.
-
+        :param profile: Comma-separated list of transportation modes that the user is willing to use.
+        :param intervals: Time ranges to calculate isochrones for. In seconds or meters, depending on `interval_type`.
+        :param interval_type: Only for compatibility. This isn't used, only 'time' is allowed.
+        :param date_time: Departure date and time (timezone aware). The default value is now (UTC).
+        :param date_time_type: Only for compatibility. This isn't used, only "depart_at" isochrones are allowed.
         :arrive_by: Set to False when searching from the location and True when searching to the
             location. Default value: False.
-        :type arrive_by: bool
-
         :param dry_run: Print URL and parameters without sending the request.
-        :param dry_run: bool
 
         :returns: An isochrone with the specified range.
         :rtype: :class:`routingpy.isochrone.Isochrones`
         """
         params = [
             ("location", convert.delimit_list(reversed(locations), ",")),
-            ("time", time.isoformat()),
+            ("time", date_time.isoformat()),
```

Comment:
and then isochrones ingest tz aware date_time strings? would be cool if the routing endpoint was capable of the same.. I was really sure OTP 2.x was a clean API :sweat_smile:

Reply:
The isochrones are in an addon. Strictly speaking, they are not OTPs. But yes, I do agree with you 😜